opentimelineio/adapters/cmx_3600.py

```diff
@@ -33,6 +33,7 @@
 # TODO: currently tracks with linked audio/video will lose their linkage when
 #       read into OTIO.
 
+import enum
```

Comment:
Python 2.7 doesn't have the `enum` module. (OTIO needs to support 2.7 and 3.5+) This came up last time we tried to use the enum module, but now I'm confused about why the Travis tests passed since it runs against both...

Reply:
Ah, yes. Thank you. The enum34 package is required by flake8, which is why `enum` is available when the tests run and there's no error. 

Are you opposed to adding enum34 as a dependency? It seems like it could be useful elsewhere in the code as well.  I'm also fine rewriting it to not use `enum`.

Reply:
Lets avoid the dependency for now. Also, maybe this explains why I'm not able to run flake8 on my machine :)

Reply:
Just put up a revision that removes the `enum` dependency